cluster/vagrant/util.sh

```diff
@@ -116,6 +167,9 @@ function create-provision-scripts {
 }
 
 function verify-cluster {
+  # TODO: How does the user know the difference between "tak[ing] some
+  # time" and "loop[ing] forever"? Can we give more specific feedback on
+  # whether "an error" has occurred?
```

Comment:
This is a common problem with all the Salt based providers (including GCE, AWS, Azure).  I will say that the move to do more as pods and less in Salt may make this better over time, and less of a concern.